.github/workflows/ecli.yaml

```diff
@@ -46,7 +92,23 @@ jobs:
     - name: make minimal ecli
       run:  make -C ecli/minimal
 
-    # - name: run unit tests
-    #   run: |
-    #     cd ecli
-    #     make test
+    - name: Package
+      if:   github.event_name == 'push' && github.ref == 'refs/heads/master'
+      shell: bash
+      run: |
+        mkdir release
+        cp ecli/target/release/ecli ./release/
+        cd release
+        tar czvf ./ecli-${{ matrix.target }}-${{ steps.set_version.outputs.result }}.tar.gz ecli
+
+    - name: Publish
+      if:   github.event_name == 'push' && github.ref == 'refs/heads/master'
+      uses: softprops/action-gh-release@v1
+      with:
+          files: |
```

Comment:
Please add a binary release for x86 like in https://github.com/eunomia-bpf/eunomia-bpf/releases/download/v0.3.3/ecc.

The total release files should contains:
- ecc binary
- ecli binary
- ecli-${{ matrix.target }}-${{ steps.set_version.outputs.result }}.tar.gz
- ecc-${{ matrix.target }}-${{ steps.set_version.outputs.result }}.tar.gz
- source code

Reply:
> Please add a binary release for x86 like in https://github.com/eunomia-bpf/eunomia-bpf/releases/download/v0.3.3/ecc.
> 
> The total release files should contains:
> - ecc binary
> - ecli binary
> - ecli-${{ matrix.target }}-${{ steps.set_version.outputs.result }}.tar.gz
> - ecc-${{ matrix.target }}-${{ steps.set_version.outputs.result }}.tar.gz
> - source code

Added now

https://github.com/oluceps/eunomia-bpf/releases/tag/v0.0.10